resolwe_bio/processes/workflows/wgbs.yml

```diff
@@ -47,6 +48,75 @@
         performance when processing a BAM file in which
         variable-length data (read, qualities, tags) do not
         otherwise need to be decoded. Default is STRICT.
+    - name: illuminaclip
+      label: Illuminaclip
```

Comment:
The label should `Adapter trimming`. `Illuminaclip` is a subcommand name of Trimmomatic, and it doesn't say much to the layperson.

Reply:
Fixed. I have also modified the description to match this label.